frontends/web/src/api/account.ts

```diff
@@ -317,4 +317,8 @@ export const addAccount = (coinCode: string, name: string): Promise<TAddAccount>
     coinCode,
     name,
   });
+};
+
+export const testRegister = (pin: string): Promise<any> => {
```

Comment:
It doesn't matter, but return type should be `Promise<null>`.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/88d28497cb56f05d63e6ea465201da1934a6fc38/backend/handlers/handlers.go#L681-L692